packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -1383,7 +1383,7 @@ export class EditableGrid extends PureComponent<EditableGridProps, EditableGridS
 
         if (showAsTab) {
             const bulkDisabled = selected.size === 0;
-            const gridDisabled = editorModel.rowCount > maxRows;
+            const showGrid = editorModel.rowCount > maxRows;
```

Comment:
isn't it that we hide the grid if the rowcount is greater than the the maxRows?
```suggestion
            const hideGrid = editorModel.rowCount > maxRows;
```

Reply:
Picky, picky. 😀